readme.md

```diff
@@ -1,6 +1,7 @@
 ![image](https://user-images.githubusercontent.com/7620947/119247309-b69d1580-bb5e-11eb-9d81-4495dfc45f21.png)
 
 [![tests](https://github.com/rodrigo-brito/ninjabot/actions/workflows/ci.yaml/badge.svg)](https://github.com/rodrigo-brito/ninjabot/actions/workflows/ci.yaml)
+[![codecov](https://codecov.io/gh/sudiptog81/ninjabot/branch/main/graph/badge.svg?token=VfrjGMzXC5)](https://codecov.io/gh/sudiptog81/ninjabot)
```

Comment:
```suggestion
[![codecov](https://codecov.io/gh/rodrigo-brito/ninjabot/branch/main/graph/badge.svg)](https://codecov.io/gh/sudiptog81/ninjabot)
```

Reply:
Ah, good catch, was just testing it out. Fixing in next commit.